pydis_core/utils/paste_service.py

```diff
@@ -80,21 +102,13 @@ async def send_to_paste_service(
 
         _lexers_supported_by_pastebin[paste_url] = list(response_json)
 
-    if lexer not in _lexers_supported_by_pastebin[paste_url]:
-        raise PasteUnsupportedLexerError(f"Lexer '{lexer}' not supported by pastebin.")
-
-    contents_size = len(contents.encode())
-    if contents_size > max_size:
-        log.info("Contents too large to send to paste service.")
-        raise PasteTooLongError(f"Contents of size {contents_size} greater than maximum size {max_size}")
+    for file in files:
```

Comment:
I think doing this would complicate things, as we'd still need to indicate to the user that some of the files failed, and then the client would need to figure out what to do with that info and how to show it to the user.

Reply:
Fair enough :D